assets/src/infrastructure/TopLevelProviders/Apollo.js

```diff
@@ -0,0 +1,25 @@
+import { ApolloClient } from 'apollo-client';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+import { HttpLink } from 'apollo-link-http';
+import { ApolloProvider } from '@apollo/react-hooks';
+
+const { graphqlEndpoint, eejsdata: { data } } = window;
```

Comment:
This is lost somewhere... That data not being set in PHP I guess.

Reply:
Thanks, this is why I made it a WIP, because I knew that some details might be missing.